tutorials/plot_artifacts_correction_ica.py

```diff
@@ -174,13 +187,19 @@
 # The idea behind corrmap is that artefact patterns are similar across subjects
 # and can thus be identified by correlating the different patterns resulting
 # from each solution with a template. The procedure is therefore
-# semi-automatic. Corrmap hence takes at least a list of ICA solutions and a
-# template, that can be an index or an array. As we don't have different
-# subjects or runs available today, here we will fit ICA models to different
-# parts of the recording and then use as a user-defined template the ICA
-# that we just fitted for detecting corresponding components in the three "new"
-# ICAs. The following block of code addresses this point and should not be
-# copied, ok?
+# semi-automatic. :func:`mne.preprocessing.ica.corrmap` hence takes a list of
```

Comment:
this will not link properly because `corrmap` does not show up in our API docs:

http://mne-tools.github.io/dev/python_reference.html

It should be added there, probably under the `mne.preprocessing` namespace, then the a link to mne.preprocessing.corrmap should work.